.github/CODEOWNERS

```diff
@@ -521,7 +521,7 @@
 /core/templates/pages/splash-page/ @jameesjohn @vojtechjelinek
 /core/templates/pages/teach-page/ @jameesjohn @vojtechjelinek
 /core/templates/pages/thanks-page/ @jameesjohn @vojtechjelinek
-
+/core/templates/pages/welcome-guide-page/ @kmaraj @pmets @sm6957
```

Comment:
@seanlip, I'm not sure whether we expect "non-members" to be the code owners. Do we have any general principle for who can be the code owner for new files? (Considering we cannot assign them to PRs/issues)

Reply:
Let's have the codeowners for this be the same as the line above. Thanks for flagging!

Reply:
fixed!